src/sandstorm/supervisor.capnp

```diff
@@ -274,15 +256,28 @@ struct ApiTokenOwner {
 
       introducerIdentity @9 :Text;
       # The identity ID through which a user's powerbox action caused the grain to receive this
-      # token. This is the identity against which the `requiredPermissions` parameter
-      # to `restore()` will be checked. This field is only intended to be filled in by the
-      # front-end during a powerbox request; a regular `save()` call produces a capability that
-      # has no "introducer".
+      # token. The token will become invalid if this identity loses access to the grain. This field
```

Comment:
> Why is it OK to loosen them somewhat but not completely?

I was somehow under the impression that any apps that do use the powerbox today would not yet need full support for `grain.requiredPermissions`. But apparently that's not the case even for the feature key vendor app.

I agree that we should refuse to restore such tokens.